workspace/api/openfdd_bridge/host_stats.py

```
@@ -0,0 +1,266 @@
+
+    cpu: dict[str, Any] = {
+        "logical_cores": _logical_cpu_count(),
+        "usage_percent": cpu_pct,
+    }
+    if load:
+        cpu["load_1"] = load[0]
+        cpu["load_5"] = load[1]
+        cpu["load_15"] = load[2]
+
+    disks: list[dict[str, Any]] = []
+    for label, path in (
```

Ruff (0.15.14)

[warning] Loop control variable `label` not used within loop body

Rename unused `label` to `_label`

(B007)